src/Sulu/Component/Webspace/Analyzer/Attributes/RequestAttributes.php

```diff
@@ -51,6 +51,6 @@ public function getAttribute($name, $default = null)
      */
     public function merge(RequestAttributes $requestAttributes)
     {
-        return new self(array_merge($this->attributes, $requestAttributes->attributes));
+        return new self(array_merge($requestAttributes->attributes, $this->attributes));
```

Comment:
This change is required because the `ParameterRequestProcessor` is called before the `WebsiteRequestProcessor`, and the values from the first one should not be overridden. However, I am not quite sure yet if this change is also ok with the `CustomUrlRequestProcessor`. But it looks that way, since the other `RequestProcessor` seems to find the same `PortalInformation`.

@wachterjohannes Can you confirm?